packages/app/src/app/pages/NewDashboard/Content/routes/Settings/NewTeam.tsx

```diff
@@ -17,14 +17,26 @@ export const NewTeam = () => {
   const onSubmit = async event => {
     event.preventDefault();
     const teamName = event.target.name.value;
-    console.warn(teamName, 'created by', user.username);
-    setLoading(true);
-    try {
-      await dashboard.createTeam({ teamName });
-      setLoading(false);
-      history.push(dashboardUrls.teamInvite(activeTeam));
-    } catch {
-      setLoading(false);
+    if (teamName && teamName.trim()) {
```

Comment:
Are changes needed here too?

Reply:
What changes are you referring to here ?